action/tx_container.go

```diff
@@ -144,7 +144,21 @@ func (etx *txContainer) ProtoForHash() *iotextypes.ActionCore {
 }
 
 func (etx *txContainer) Proto() *iotextypes.ActionCore {
-	panic("should not call txContainer's Proto()")
+	if len(etx.raw) == 0 {
```

Comment:
according to https://github.com/iotexproject/iotex-core/pull/4431/files#diff-ba24a3dbceb688932c90adba65e95c51240bfb910184cb70fde9d8e18474aa51R175-R177, raw cannot be empty